json-endpoints/jetpack/class.jetpack-json-api-get-comment-backup-endpoint.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class Jetpack_JSON_API_Get_Comment_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/comments/%d/backup      -> $blog_id, $comment_id
+
+	protected $needed_capabilities = array();
```

Comment:
might be good to leave a comment here why we are not checking for capabilities. So that it is clear.